apps/system/locales/system.en-US.properties

```diff
@@ -477,7 +477,10 @@ payment-flow=Purchase
 
 # Remote Debugger Connection Dialog
 remoteDebuggerPromptUSB=Allow USB debugging connection?
-remoteDebuggerPromptTCP=Allow remote debugging connection from {{host}}:{{port}}?
+remoteDebuggerPromptTCP.innerHTML=Allow remote debugging connection from {{host}}:{{port}}?<br/><br/>Client ID: {{cert}}
```

Comment:
Note: Please also use "SHA-256 certificate:" or "My SHA-256 certificate:" in WebIDE, instead of "My Cert:".